services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -51,6 +51,7 @@ import accordion from "verbiage/pages/accordion";
 // assets
 import arrowLeftIcon from "assets/icons/icon_arrow_left_blue.png";
 import alertIcon from "assets/icons/icon_alert_circle.png";
+import { AddNewWorkPlanModal } from "components/modals/AddNewWorkPlanModal";
```

Comment:
consistency suggestion: if you add `AddNewWorkPlanModal` to the `src/components/index.ts` file you can import it with the other "components" imports above